vault/server.sls

```diff
@@ -22,23 +22,15 @@ generate self signed SSL certs:
     - group: root
     - mode: 755
 
-/etc/vault/config:
-  file.directory:
-    - user: root
-    - group: root
-    - mode: 755
-    - require:
-      - file: /etc/vault
-
-/etc/vault/config/server.hcl:
-  file.managed:
-    - source: salt://vault/files/server.hcl.jinja
-    - template: jinja
+/etc/vault/server.json:
```

Comment:
In fact the file is serialised in JSON, but I understand your point, I will update accordingly.